### PR TITLE
OAEWDGT-94 - Truncate review

### PR DESCRIPTION
**Pull request by christianvuerings**
Jira: https://jira.sakaiproject.org/browse/OAEWDGT-94

### CODE REVIEW

File: app/views/core/_review.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="widget_review">
     <%= link_to review.user.name, user_path(:id => review.user.id, :url_title => review.user.url_title), :class => "wl-regular-link wl-bold" %>
-    <p>"<%= review.review %>"</p>
+    <p data-completetext="<%= review.review %>"><%= raw truncate(review.review, :length => 250, :separator => ' ', :omission => ' <button class="review_read_more wl-link-button">read more</button>') %></p>
```
Comment (croby) [inline app/views/core/_review.html.erb]: We probably don't want to `raw` the whole thing, as that could expose xss issues -- can we just `raw` the omission result?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -92,10 +92,16 @@ $(function() {
         return false;
     }
 
+    var readmore = function() {
+        var parentel = $(this).parent();
```
Comment (croby) [inline app/assets/javascripts/application.js]: can we name this `$parentEl` for readability (and jQuery-ability)?

Comment (christianvuerings) [inline app/assets/javascripts/application.js]: I'll make it `$parentElement`

File: app/views/core/_review.html.erb (hunk 1 above)
Comment (christianvuerings) [inline app/views/core/_review.html.erb]: Added a helper method for this

File: app/helpers/widget_helper.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# encoding: utf-8
+
+module WidgetHelper
+
+  def review_description review
```
Comment (croby) [inline app/helpers/widget_helper.rb]: Per convention, let's make this

``` ruby
def review_description(review)
```

File: app/helpers/widget_helper.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    output = h truncate(review.review, :length => 250, :separator => ' ', :omission => ' ')
+    output += button_tag("read more", :type => 'button', :class => 'review_read_more wl-link-button') if review.review.size > 250
+    output.html_safe
+  end
+
+end
\ No newline at end of file
```
Comment (croby) [inline app/helpers/widget_helper.rb]: Let's add a newline here so git gets off our back

File: app/helpers/widget_helper.rb (hunk 3 above)
Comment (christianvuerings) [inline app/helpers/widget_helper.rb]: Wasn't aware of this.
It should be fixed now.

File: app/helpers/widget_helper.rb (hunk 4 above)
Comment (christianvuerings) [inline app/helpers/widget_helper.rb]: Ok

**Pull request merged**